en/docs/guides/user-mgt/delete-users.md

```diff
@@ -0,0 +1,111 @@
+# Delete an existing user 
+
+## Delete a user using the admin portal 
+
+{!fragments/insert-link!}
+
+---
+
+## Delete a user using SCIM 
+
+**Request**
+
+```curl
+curl -v -k --user {IS_USERNAME}:{IS_PASSWORD} -X DELETE https://{IS_IP}:{IS_PORT}/wso2/scim/Users/{SCIM_USER_ID} -H "Accept: application/json"
+```
+
+**Sample Request**
+
+```curl 
+curl -v -k --user admin:admin -X DELETE https://localhost:9443/wso2/scim/Users/b228b59d-db19-4064-b637-d33c31209fae -H "Accept: application/json"
+```
+
+You receive a response with status `200 OK` and the user will be deleted from the user store.
+
+---
+
+## Delete a user using SOAP 
+
+You can also delete a user by calling the `RemoteUserStoreManager` service . If you are new to admin services, see [Calling Admin Services](insert-link).
+
+The following SOAP method will give you the user ID of the relevant username. 
+
+**Request Sample**
+
+```
+<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope" xmlns:ser="http://service.ws.um.carbon.wso2.org">
+   <soap:Header/>
+   <soap:Body>
+      <ser:deleteUser>
+         <!--Optional:-->
+         <ser:userName>user123</ser:userName>
+      </ser:deleteUser>
+   </soap:Body>
+</soap:Envelope>
+```
+
+---
+
+## Tracking user deletion on deleting a user (optional)
+
+WSO2 Identity Server (WSO2 IS) allows you to track details related to
+user deletion by writing the following information to a log file each
+time a user is deleted:
+
+-   The user name of the deleted user.
+-   The user store domain name of the deleted user.
+
+-   The tenant domain name of the deleted user.
+-   The tenant ID of the deleted user.
+-   The timestamp that the user was deleted.
+
+You can use this log file as an input to the Identity Anonymization tool
+that is packaged with WSO2 Identity Server, to ensure that you [remove references to all deleted user identities](insert-link).
+
+Follow the steps below to configure WSO2 Identity Server to log details
+related to user deletion each time you delete a user:
+
+1.  Add the following property to the 
+    `           <IS_HOME>/repository/conf/deployment.toml          `
+    file, and set it to
+    `           true          `
+
+    ``` toml
+    [event.default_listener.user_deletion]
+    priority= "98"
+    enable = true 
+    ```
+
+2.  Add the 
+    `           <IS_HOME>/repository/conf/deployment.toml          `
+    file, and set `           enable          ` to
```

Comment:
please check this step? It seems like its saying to add the deployment file?

Reply:
Done